pkg/apis/multiclusterdns/v1alpha1/dnsendpoint_types.go

```diff
@@ -30,17 +30,17 @@ type TTL int64
 // it is then stored in a persistent storage via serialization
 type Labels map[string]string
 
-// Endpoint is a high-level way of a connection between a service and an IP
+// Endpoint is a high-level connection between a service and an IP.
```

Comment:
I would consider changing `connection` to `association` or `mapping`. When I see connection, I think of network plumbing.

Reply:
Good point